### ISSUE TITLE
Transaction not complete

### ISSUE THREAD

**Issue by LM-Launch14** [ethereum/mist#3696]
- [ ] I've asked for help in the [Mist Gitter](http://gitter.im/ethereum/mist) before filing this issue.

```
Version: `10.13.1 `
OS & Version: osx
Node version: `geth 1.7.@` 
Number of blocks synchronized: Downloading block 5,109,921 of 5,109,997
```

https://etherscan.io/tx/0x2807cf2b7516eb16cf0cdd4c12fa6c2460e5b0ad4af2a1975d1f8d804e04ce58   
13 hours and counting ```Transaction not complete```   
![screen shot 2018-02-17 at 9 03 20 pm](https://user-images.githubusercontent.com/36553779/36347528-aa7101d2-1426-11e8-9645-f21a62748550.png)

**Comment by bbtech1** [comment]
the same  problem. 4 transaction

**Comment by LM-Launch14** [comment]
Transaction still has not completed.

Robert
330-995-0864

**Comment by OpenJeDi** [comment]
I have the same issue - on the Rinkeby test network.

I called a method from a contract (with 0 gas price, which I doesn't seem to be able to change). The transaction goes through, but gets no confirmations and does not show up in etherscan.

**Comment by ryanio** [comment]
Hey all, we have released a new version (`0.11.0`) that has better syncing features - please give it a try. If you have any new problems that you can reproduce, please create a new issue with the details on how we can help. Thanks!
